selectors/selectors.md

```diff
@@ -199,38 +204,40 @@ The shape of a block could look like this (in JSON):
 If you know you want five parents you could use Path Selectors:
 
 ```json
-{
-  "cidRootedSelector": {
-    "root": "cidabcdef",
-    "selectors": [
-      {"selectPath": "parent"},
-      {"selectPath": "parent"},
-      {"selectPath": "parent"},
-      {"selectPath": "parent"},
-      {"selectPath": "parent"}
-    ]
-  }
-}
+{"selectFields":{"parent":
+	{"selectFields":{"parent":
+		{"selectFields":{"parent":
+			{"selectFields":{"parent":
+				{"selectFields":{"parent":
+					true}}}}}}}}}}
 ```
 
-But there's also a better way with using a Recursive Selector with a limit:
+This selector matches the fivth-deepest "parent" (and in the context of
+graphsync or other merkleproof applications, will yield all five nodes).
+
+But this gets a bit verbose.  We can explore the same tree in a similar
+pattern with another mechanism -- recursive exploration:
 
 ```json
-{
-  "cidRootedSelector": {
-    "root": "cidabcdef",
-    "selectors": [
-      {"selectRecursive": {
-        "follow": [
-          {"selectPath": "parent"}
-        ],
-        "depthLimit": 5
-      }}
-    ]
-  }
-}
+{"selectRecursive": {
+  "depthLimit": 5,
+  "next":
+    {"selectFields":{"parent":
+      true}}
+}}
 ```
 
+This will traverse the same set of nodes as the previous example -- however,
+it has has a *slightly* different effect!
+
+Using a recursive selector in this way matches *each* of the "parent" nodes,
```

Comment:
Can we use a different property value than “parent.” It’s not clear if this refers to a property because parent is such a broadly used term.